src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig

```diff
@@ -57,82 +64,167 @@
         </tr>
       </thead>
       <tbody>
-        {% for product in orderForViewing.products.products %}
-          <tr>
-            <td>
-              {% if product.imagePath %}
-                <img src="{{ product.imagePath }}" alt="{{ product.name }}">
-              {% endif %}
-            </td>
-            <td>
-              <a href="{{ path('admin_product_form', {'id': product.id}) }}">
-                <p class="mb-0">{{ product.name }}</p>
-                {% if product.reference %}
-                  <p class="mb-0">
-                    {{ 'Reference number:'|trans({}, 'Admin.Orderscustomers.Feature') }}
-                    {{ product.reference }}
-                  </p>
-                {% endif %}
-                {% if product.supplierReference is not empty %}
-                  <p class="mb-0">
-                    {{ 'Supplier reference:'|trans({}, 'Admin.Orderscustomers.Feature') }}
-                    {{ product.supplierReference }}
-                  </p>
-                {% endif %}
+        {% for product in orderForViewing.products.products|slice(0, paginationNum) %}
+          {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/product.html.twig' with {'product': product} %}
+        {% endfor %}
+        <tr id="addProductTableRow" class="d-none">
+          <td colspan="2" class="pr-5">
+            <input type="hidden" id="addProductId">
+            <input type="hidden" id="addProductTaxRate">
+            <label for="addProductSearch">{{ 'Add a product'|trans({}, 'Admin.Orderscustomers.Feature') }}</label>
+            <input type="text" class="form-control" id="addProductSearch" placeholder="{{ 'Search for a product'|trans({}, 'Admin.Orderscustomers.Feature') }}">
+            <div class="input-group mt-2 d-none" id="addProductCombinations">
```

Comment:
We should try to avoid the use of mt-* classes and other modifiers in the future, as we said with Pablo thats the contrary of the CSS goal and it remains like using a style attribute, we need to speak about this to find a good way to do this

Reply:
I don't understand. We use Bootstrap for the BO and not having CSS style which override bootstrap. If we don't use mt-*, we need to have supplementary CSS. It is not logical.

Reply:
I guess he means html should be all about semantic not design, `mt-2` includes the value of the margin. Whereas this value should be in a css targeting `#addProductCombinations` (ideally)
Modifiers like `mt-2` are useful but if you want to make a BO theme then you have to override all those hard coded style classes

Reply:
No, you can use SCSS variable for creating your BO Theme. Bootstrap rocks for it.

Reply:
Warning : `mt-2` is not always equals to 2px, but it's equal to `$spacer * .5` from the Bootstrap Documentation. So if you use `m*-*` everywhere, your interface have sense. But if you override one `mt-2`, you can break your interface.